AlfrescoSDK/AlfrescoSDK/Utils/AlfrescoConnectionDiagnosticUtils.h

```diff
@@ -0,0 +1,28 @@
+/*
+ ******************************************************************************
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of the Alfresco Mobile SDK.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *****************************************************************************
+ */
+
+#import <Foundation/Foundation.h>
+
+@interface AlfrescoConnectionDiagnosticUtils : NSObject
+
++ (NSDictionary *) createDictionaryForStartEventForEventName:(NSString *)eventName;
++ (NSDictionary *) changeDictionaryForEndEvent:(NSDictionary *) dict isSuccess:(BOOL) isSuccess withError:(NSError *)error;
```

Comment:
We tend to avoid using ```with``` when referring to additional parameters in a method signature.

```+ (NSDictionary *)changeDictionaryForEndEvent:(NSDictionary *)dict isSuccess:(BOOL)isSuccess error:(NSError *)error```